Formula/s/stellar-cli.rb

```diff
@@ -0,0 +1,32 @@
+class StellarCli < Formula
+  desc "Stellar command-line tool for interacting with the Stellar network"
+  homepage "https://developers.stellar.org"
+  url "https://github.com/stellar/stellar-cli.git",
+      tag:      "v21.3.0",
+      revision: "11746937e813346272faf68e70cbdbc1b0df780f"
```

Comment:
Do we need a git checkout? Can we use a tarball instead?

Reply:
```suggestion
  url "https://github.com/stellar/stellar-cli/archive/refs/tags/v21.3.0.tar.gz"
  sha256 "3bbd1eba92ca5f6dd451a8606520b40cc71e2796b0efa32303c52a7901a2a944"
```